### DISCUSSION TITLE
need i set a new branch?

### DISCUSSION THREAD

**Discussion by ZIHAOJIA** [MichaelKelly9/Tafe24S1T104_Ext#5] [General]
when i finish the code ,i realize i modify it on dev branch ,when I want to commit it ,should it creat a new branch named feature2_mortgage_calc?

**Comment by MichaelKelly9** [comment]
Yes we needed to clone the repo, checkout the Dev branch and then create a new feature branch locally. Once coded, commit to feature branch and then push feature branch into GitHub Repo.
